src/pages/centers.tsx

```diff
@@ -567,6 +761,65 @@ const Center: React.FC = () => {
           isEditModal={true}
           userId={userId}
         />
+
+        <SimpleModal
+          open={isEditForm}
+          onClose={onCloseEditForm}
+          showFooter={false}
+          modalTitle={t("COMMON.UPDATE_CENTER")}
+        >
+          {schema && uiSchema && (
+            <DynamicForm
+              schema={schema}
+              uiSchema={uiSchema}
+              onSubmit={handleUpdateAction}
+              onChange={handleChangeForm}
+              onError={handleError}
+              widgets={{}}
+              showErrorList={true}
+              customFields={customFields}
+              formData={editFormData}
+              id="xyz"
```

Comment:
@upendraTekdi why id is xyz here

Reply:
done

Reply:
we are using id for conditional use of dynamicForm buttons. if we are passing id="dynamic-form" then it will use dynamicForm rjsf buttons otherwise it will use passed children buttons.